src/js/apps/patients/worklist/filters_state.js

```diff
@@ -0,0 +1,29 @@
+import { sortBy, isEqual, omit, size, isNull } from 'underscore';
+
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+export default Backbone.Model.extend({
+  preinitialize() {
+    this.currentOrg = Radio.request('bootstrap', 'currentOrg');
+    this.states = this.currentOrg.getStates();
+  },
+  getDefaultStatesFilter() {
+    const isDoneOnly = this.get('worklistId') === 'done-last-thirty-days';
```

Comment:
This is where the `state.filters.worklistId` is used. Without it, the incorrect filters count is displayed and the incorrect default state is added after `Clear Filters` for the `Done < 30 Days` list.

This state doesn't have access to the `this.id` worklist id that resides in the `worklist_state.js` file.